### ISSUE TITLE
Changes to openmc API 

### ISSUE THREAD

**Issue by shimwell** [fusion-energy/openmc-plasma-source#71]
[This PR](https://github.com/openmc-dev/openmc/pull/2214) changes ```Muir``` to ```muir``` in the python API for openmc so we will need some changes on this package

**Comment by RemDelaporteMathurin** [comment]
If I understand the Muir class is going to be replaced by a muir function returning a Normal class.

**Comment by shimwell** [comment]
Yep that is right, we will just need to replace ```Muir``` with ```muir``` in the code.

When v0.14.0 gets released use if ```Muir``` will still work but print a warning message
